.github/workflows/acceptance.yaml

```diff
@@ -0,0 +1,34 @@
+name: Acceptance Tests
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch: {}
+
+jobs:
+  kind:
+    strategy:
+      fail-fast: false
+      matrix:
+        kind-k8s-version: [1.14.10, 1.19.11, 1.22.4]
```

Comment:
The thinking here is 1.14 is the minimum version, 1.22 is the max, and I wanted to include an in-between version that was pre-1.21 but more recent than 1.14.

Reply:
How about 1.14 and the three most recent versions? Then it includes our oldest version + roughly approximates the support windows for major managed Kubernetes providers [1](https://cloud.google.com/kubernetes-engine/docs/release-notes) [2](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html) [3](https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#kubernetes-version-support-policy). Kubernetes releases every 3 months and each version is [supported for 12 months](https://kubernetes.io/blog/2020/08/31/kubernetes-1-19-feature-one-year-support/), so maybe even the 4 most recent versions would make sense?

Reply:
Yeah, that makes sense. I went with 1.14 and the 4 most recent minor versions. It turns out kind node images aren't published for every patch release, so I used the latest patches I could find for each in e29aae9.